spec/services/google_scraping_service_spec.rb

```diff
@@ -0,0 +1,164 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe GoogleScrapingService, type: :service do
+  include ActiveJob::TestHelper
+
+  describe '#call' do
+    context 'given valid keyword (with top position adwords)' do
+      it 'updates status to completed' do
+        user = Fabricate(:user)
+        keyword = Fabricate(:keyword, user_id: user[:id], keyword: 'AWS')
+
+        VCR.use_cassette('with_top_position_adwords', record: :none) do
+          GoogleScrapingJob.perform_now(keyword.id, keyword.keyword)
```

Comment:
we are testing the `GoogleScrapingService` why do we need to rely on the `GoogleScrapingJob` 🤔 

As it's unit test, the test should be really unit, focus to the `main class` as we are testing...

Reply:
Fixed in 84ff270ffc1f3cd5d3700746e3ca3e9043558652